packages/block-library/src/post-comments/edit.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { useEntityId } from '@wordpress/core-data';
+
+function PostCommentsDisplay( { postId } ) {
+	return useSelect(
+		( select ) => {
+			const comments = select( 'core' ).getEntityRecords( 'root', 'comment', {
+				post: postId,
+			} );
+			return (
+				comments &&
```

Comment:
When there are no comments we are displaying nothing, show we display some message saying there are no comments?

Reply:
I think so. We still need designs for this. It should also be editable (the message).

Reply:
Maybe some kind of dummy placeholder for now so this can be merged and worked on later?

Reply:
Added a dummy placeholder :)